projects/ion/src/lib/picker/control-picker/control-picker.component.ts

```diff
@@ -0,0 +1,114 @@
+import { Component, Input, Output, EventEmitter } from '@angular/core';
+
+export type TypeEvents =
+  | 'previousYear'
+  | 'previousMonth'
+  | 'nextMonth'
+  | 'nextYear'
+  | 'changeMonth'
+  | 'changeYear';
+
+type Events = {
+  type: TypeEvents;
+  value?: string;
+};
+
+export interface ControlEvent {
+  event?: Events;
+}
+
+export interface ControlPickerComponentProps {
+  month: string;
+  year: string;
+  controlPickerEvent?: EventEmitter<ControlEvent>;
+}
+
+@Component({
+  selector: 'ion-control-picker',
+  templateUrl: './control-picker.component.html',
+  styleUrls: ['./control-picker.component.scss'],
+})
+export class ControlPickerComponent {
+  @Input() month: string;
+  @Input() year: string;
+  @Output() controlPickerEvent: EventEmitter<ControlEvent> =
+    new EventEmitter<ControlEvent>();
+
+  months = [
+    'Jan',
+    'Fev',
+    'Mar',
+    'Abr',
+    'Mai',
+    'Jun',
+    'Jul',
+    'Ago',
+    'Set',
+    'Out',
+    'Nov',
+    'Dez',
+  ];
+  showContainerToChooseTheMonth = false;
+  showContainerToChooseTheYear = false;
+  rangeYear: number[] = [];
+
+  handlePreviousYear(): void {
+    this.controlPickerEvent.emit({ event: { type: 'previousYear' } });
+  }
+
+  handlePreviousMonth(): void {
+    this.controlPickerEvent.emit({ event: { type: 'previousMonth' } });
+  }
+
+  handleNextMonth(): void {
+    this.controlPickerEvent.emit({ event: { type: 'nextMonth' } });
+  }
+
+  handleNextYear(): void {
+    this.controlPickerEvent.emit({ event: { type: 'nextYear' } });
+  }
+
+  toogleVisibleContainerToChooseTheMonth(visible: boolean): void {
```

Comment:
```suggestion
  setVisibleContainerToChooseTheMonth(visible: boolean): void {
```
Since you're setting the visibility, not toggling, I think it would be best if the function have a more appropriate name too.